imports/eosio.imports.in

```diff
@@ -194,3 +194,4 @@ confirm_bw_limits
 get_ram_provider
 update_stake_proxied
 recall_stake_proxied
+confirm_approve_ram
```

Comment:
I think this method should be called `confirm_ram_usage`, without `approve`. So, it will be the same with `approvebw` with `confirm_bw_limits` and we will have the following names in the call chain: `requestram` -> `approveram` -> `confirm_ram_usage`